src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="site">
+	<div class="light site">
```

Comment:
if you make the computed prob called theme that returns the class name you can do `:class="theme"` so it looks like
```html
<div class="site" :class="theme">
```